src/tests/Common/helixpublishwitharcade.proj

```diff
@@ -85,9 +85,9 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <BinDir>$([MSBuild]::NormalizeDirectory($(TestWorkingDir)))</BinDir>
-    <CoreRootDirectory>$(BinDir)Tests\Core_Root\</CoreRootDirectory>
-    <PayloadsRootDirectory>$(BinDir)Payloads\</PayloadsRootDirectory>
+    <TestBinDir>$([MSBuild]::NormalizeDirectory($(TestWorkingDir)))</TestBinDir>
```

Comment:
Where is `TestWorkingDir`? Why not just change that to `TestBinDir`?

Reply:
Good point, fixed in 2nd commit.